scripts/archive.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+source  /etc/profile.d/modules.sh
+
+set -a
+
+module purge
+module load pbs
+module use /g/data/hh5/public/modules
+module use ~access/modules
+module load cdo
+module load nco
+module load pythonlib/um2netcdf4/2.0
+module load conda/analysis3
+
+# set the environment variables for ACCESS-Archiver
+loc_exp=${PWD##*/}
+subdaily=false
+access_version=esmpayu
+arch_dir=/g/data/$PROJECT/$USER/archive/access-esm
+base_dir=/scratch/$PROJECT/$USER/access-esm/archive
+comp_proj=$PROJECT
+here=/g/data/tm70/kr4383/ACCESS-Archiver 
+
+plev8=false
+zonal=false
+convert_unknown=false
+ncexists=false
+arch_grp=$PROJECT
+UMDIR=/projects/access/umdir
```

Comment:
Should probably use a consistent approach for referring to these directories. Above it was `~access`, which resolves to `/g/data/access/projects/access` as does `/projects/access`
```
$ ls -l /projects/access
lrwxrwxrwx. 1 root root 30 Dec 16  2020 /projects/access -> /g/data/access/projects/access
```
and define it once in one place at the top and refer to the same variable when you need this path.